script/generate-mime.rb

```diff
@@ -97,7 +98,7 @@ def get_matches(parent)
 puts "  # :nodoc:"
 puts "  EXTENSIONS = {"
 extensions.keys.sort.each do |key|
-  puts "    '#{key}' => '#{extensions[key]}',"
+  puts "    '#{key}' => ['#{extensions[key].join('\',\'')}'],"
 end
```

Comment:
It seems you always wrap it in an array? This means the above check for Array or the wrapping in [*mime] as I wrote it is unnecessary.